API.md

```diff
@@ -84,6 +90,20 @@ This will open a browser window that will allow you to authenticate in.
 ##### Response
 The bearer token should be returned and viewable in the browser window after authenticating.
 
+#### iRODS as an OAuth Protected Resource
+To run as an OAuth protected resource, set `mode` to `protected_resource`.
```

Comment:
let's make this sentence look more similar to the above...

"set `mode` in the `openid_connect` stanza to `protected_resource `."

Reply:
Done.